### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by peidlmatyas**
Fixed the errrors in TorpedoStore that SonarLint found

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +31,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (Ralle001) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Why did you change it from =- to -=?

**Review by Ralle001 — CHANGES_REQUESTED** [review]

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (peidlmatyas) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: added comment for clarification

**Review by Ralle001 — APPROVED** [review]
